### PR TITLE
Added iframe-resizer

### PR DESCRIPTION
**Pull request by brandomeniconi**

### CODE REVIEW

File: files/iframe-resizer/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "iframe-resizer",
+  "repo": "davidjbradshaw/iframe-resizer",
+  "files": {
+    "basePath": "js"
```
Comment (jimaek) [inline files/iframe-resizer/update.json]: Add an `include` parameter with `*.min.*`
We dont need all the files in js/ for production

**Comment by brandomeniconi** [comment]
sorry, I'm newbie here, included only production files as you suggested.

File: files/iframe-resizer/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "David J. Bradshaw"
+github = "https://github.com/davidjbradshaw/iframe-resizer"
+homepage = "https://github.com/davidjbradshaw/iframe-resizer"
+description = "Keep same and cross domain iFrames sized to their content with support for window/content resizing, in page links, nesting and multiple iFrames"
+mainfile = "js/iframeResizer.min.js"
```
Comment (jimaek) [inline files/iframe-resizer/info.ini]: Just `iframeResizer.min.js` js/ will be stripped by the bot

Comment (brandomeniconi) [inline files/iframe-resizer/info.ini]: Ok so I need to change the line to ```mainfile = "iframeResizer.min.js"```, right?

Comment (jimaek) [inline files/iframe-resizer/info.ini]: Yes

Comment (brandomeniconi) [inline files/iframe-resizer/info.ini]: Done!
Thanks for first time support

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**